amf-client/shared/src/test/resources/upanddown/cycle/raml10/api-with-multi-line-schema/dumped.raml

```diff
@@ -1,7 +1,7 @@
 #%RAML 1.0
 title: api
 types:
-  mySchema: |
+  mySchema: |-
```

Comment:
Due to removing line breaks

Reply:
Is equivalent, I don't see a big issue here neigher.